assets/src/edit-story/constants.js

```diff
@@ -1,11 +1,12 @@
+export const ADMIN_TOOLBAR_HEIGHT = 32;
```

Comment:
Could we check the if the body has the `admin-bar` class on before making this 32?